internal/security/config/config/config.go

```diff
@@ -0,0 +1,137 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package config
+
+import (
+	"fmt"
+	"net/url"
+
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/config"
+)
+
+type ConfigurationStruct struct {
```

Comment:
Why do we need to duplicate proxy-setup's config when we are sharing it's config file. Should be able to import it's config package and use it's ConfigurationStruct .  Must comment why, but better than having to keep the two config files in sync.